code/TestDataObjectExtension.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\Extension;
+
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\Security\Permission;
+use SilverStripe\Security\PermissionProvider;
+
+class TestDataObjectExtension extends DataExtension implements PermissionProvider
+{
+    public function providePermissions()
+    {
+        return [
+            'TEST_DATAOBJECT_EDIT' => [
+                'name' => _t(
+                    __CLASS__ . '.EditPermissionLabel',
+                    'Edit a company'
```

Comment:
```suggestion
                    'Manage a test object'
```
It's not only companies and it's not only editing.

Reply:
DONE